src/browser.qml

```diff
@@ -96,4 +99,19 @@ ApplicationWindow {
             z: -1
         }
     }
+    
+    DisabledByMdmView {
+        enabled: !AccessPolicy.browserEnabled
+        //% "Web browsing"
+        activity: qsTrId("sailfish_browser-la-web_browsing");
+        onEnabledChanged: {
+            if (enabled) {
```

Comment:
In following scenario you'd still have content running... 
1) Browser is running on private mode
2) Browser is disabled via MDM

Reply:
Also if private mode was previously used, you'd get those tabs restored. Inserted a tab model clear after the change.